CRM/Core/Form/Task/PDFLetterCommon.php

```diff
@@ -36,10 +36,13 @@ public static function preProcess(&$form) {
   /**
    * Build the form object.
    *
+   * @deprecated
+   *
    * @var CRM_Core_Form $form
    * @throws \CRM_Core_Exception
    */
   public static function buildQuickForm(&$form) {
+    CRM_Core_Error::deprecatedFunctionWarning('no supported alternative for non-core code');
```

Comment:
Oops I merged a little too soon. The thank-you letters hit this.

Reply:
opps - well we can fix that